source/blog/042-wide-gamut-colors-in-sass.md

```diff
@@ -0,0 +1,279 @@
+
+<div style="display: grid; gap: var(--sl-gutter--quarter); grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));">
+<img src="/assets/img/blog/042-p3-srgb.png" alt="display-p3 gamut rendered in sRGB space adds unequal red and green horns to the sRGB cube" width=600 height=600 />
+<img src="/assets/img/blog/042-p3-hsl.png" alt="display-p3 gamut rendered in hsl space creates a boot-like bulge of green near the base of the hsl cylinder" width=600 height=600 />
+</div>
+
+The crisp edges and clean math of `sRGB` formats were only possible because we knew exactly what colors could be displayed, and we arranged those colors to fit perfectly into a box. But human color perception is not so clear-cut, and it doesn't align perfectly with the gamut of any monitors on the market. When we attempt to space all the same colors *evenly* based on human perception rather than simple math, we get an entirely different shape with swooping edges. This is the `display-p3` gamut in `oklch` space:
+
+<img src="/assets/img/blog/042-p3-oklch.png" alt="display-p3 gamut rendered in oklch space forms a skewed cube with a conic black base" width=600 height=600 />
+
+The practical difference is particularly noticeable when we compare colors of the same 'lightness' in `hsl` vs `oklch`. Humans perceive yellow hues as lighter than blues. By scaling them to fit in the same range, `hsl` gives us a yellow that is much brighter than the blue:
+
+<img src="/assets/img/blog/042-blue-yellow.jpg" alt="on the left a blue and much brighter yellow, on the right our yellow is much darker to match the blue tone" width=753 height=209 />
+
+## New CSS formats give us the choice
+
+Moving forward, there are two directions we could go with wide gamut colors:
+
+- Color formats that re-fit larger and larger gamuts into simple coordinates, stretching the colors to preserve clean, geometric boundaries.
+- Color formats that maintain their *perceptually uniform* spacing, without any regard for specific gamuts.
+
+On the one hand, clean boundaries allow us to easily stay inside the range of available colors. Without those boundaries, it would be easy to *accidentally* request colors that aren't even physically possible. On the other hand, we expect these colors to be *perceived* by *other humans* -- and we need to make things *look* consistent, with enough contrast to be readable.
+
+The [CSS Color Module Level 4](https://www.w3.org/TR/css-color-4/) defines a number of new CSS color formats. Some of them maintain geometric access to specific color spaces. The `hwb()` function has been around for several years now, and defines `sRGB` colors using `hue`, `whiteness`, and `blackness` channels.  It's an interesting format, and [I've written about it before](https://www.miriamsuzanne.com/2022/06/29/hwb-clamping/).
```

Comment:
The transition to talking about `hwb()` here feels a bit abrupt. Maybe a bridge sentence like "You're probably already familiar with the `rgb()` and `hsl()` functions." would help?